gui/default/syncthing/core/syncthingController.js

```diff
@@ -1097,6 +1097,134 @@ angular.module('syncthing.core')
             }
         };
 
+        $scope.deviceStatusIcon = function(cfg) {
+            switch ($scope.deviceStatus(cfg)) {
+                case 'disconnected':
+                case 'disconnected-inactive':
+                    return 'fa-power-off';
+                case 'insync':
+                    return 'fa-check';
+                case 'paused':
+                    return 'fa-pause';
+                case 'syncing':
+                    return 'fa-sync';
+                case 'unused-disconnected':
+                case 'unused-insync':
+                case 'unused-paused':
+                    return 'fa-unlink';
+
+            }
+        };
+
+        $scope.deviceStatusText = function(device) {
+            var text = '';
+            switch ($scope.deviceStatus(device)) {
+                case 'disconnected':
+                    return $translate.instant('Disconnected');
+                case 'disconnected-inactive':
+                    return $translate.instant('Disconnected (Inactive)');
+                case 'insync':
+                    return $translate.instant('Up to Date');
+                case 'paused':
+                    return $translate.instant('Paused');
+                case 'syncing':
+                    text = $translate.instant('Syncing') +
+                    ' (' +
+                    percentFilter($scope.completion[device.deviceID]._total) +
+                    ', ' +
+                    binaryFilter($scope.completion[device.deviceID]._needBytes) +
+                    'B)';
```

Comment:
That's too much line splitting for my taste.  How about this?

```suggestion
                    ' (' + percentFilter($scope.completion[device.deviceID]._total) + ', ' +
                    binaryFilter($scope.completion[device.deviceID]._needBytes) + ' B)';
```

And there should always be a space between a quantity and its unit. I'm not sure if we have any example of this (without space) anywhere?

Reply:
It was like this before… so I just made sure to use the same format. Should I add the space? It would make the whole thing use more space horizontally.

Reply:
Yes, please.  If you're already touching that line.

Reply:
Ah, so that's what the binaryFilter does.  Thanks for checking, alright to drop the space then.